### PR TITLE
[CONFIG] Add PHPCS & Travis integration.

### PR DESCRIPTION
**Pull request by anthonywnz**
Add PHPCS using the WordPress-Extra and PHPCompatibilityWP standard.
Refactor code to be compatible with this standard.

Add travis integration to run phpcs on PR's.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: php
+php:
+  - 7.3
+
+install:
+  - composer install
+
+script:
+  - "./vendor/bin/phpcs ."
```
Comment (jenkoian) [inline .travis.yml]: Missing newline at end of file.

File: classes/QM_Collector_Memcache_Stats.php [hunk 2]
```diff
@@ -1,16 +1,17 @@
 <?php
 
+declare( strict_types=1 );
```
Comment (jenkoian) [inline classes/QM_Collector_Memcache_Stats.php]: I agree we would want this, but there's no way the upstream would accept due to it locking out older versions of php, something wordpress are historically dead against. I think we shouldn't do this until we go all in on our fork (ideally they will merge your changes and we can just use the main one)

Comment (jenkoian) [reply classes/QM_Collector_Memcache_Stats.php]: Sorry I think this comment applies to entire pr.

Comment (anthonywnz) [reply classes/QM_Collector_Memcache_Stats.php]: Ah that makes sense. I'll use a different ruleset then (just added a random Automattic one to get started).

**Review by jenkoian — APPROVED** [review]

File: classes/QM_Collector_Memcache_Stats.php (hunk 2 above)
Comment (anthonywnz) [reply classes/QM_Collector_Memcache_Stats.php]: @jenkoian changed to use WordPress-Extra and PHPCompatibilityWP :+1:

**Pull request merged**